### DISCUSSION TITLE
How to escape terminal mode

### DISCUSSION THREAD

**Discussion by Patrick-web** [NvChad/NvChad#1193] [Q&A] [answered]
Is there a keymap to escape the  terminal i.e move focus back to other windows without closing it

**Comment by siduck** [comment, marked as answer]
Ctrl x

**Reply by WingDust** [reply]
how to mapping esc to escape terminal mode ?

**Reply by WingDust** [reply]
@siduck

**Reply by siduck** [reply]
@WingDust override mappings, u should know how to override default mappings. check our docs

**Reply by WingDust** [reply]
Can get a example
 ,docs example is not good  @siduck

**Reply by siduck** [reply]
which part did u read :l

**Reply by WingDust** [reply]
@siduck https://nvchad.github.io/config/Mappings#mappings-table-structure i don't know how follow this part

**Reply by siduck** [reply]
@WingDust its just the theory part, read below it :l

**Reply by WingDust** [reply]
```lua
-- Just an example, supposed to be placed in /lua/custom/

local M = {}
--M.options ,M.ui,M.mappings,M.plugins = {},{},{},{}

-- make sure you maintain the structure of `core/default_config.lua` here,
-- example of changing theme:
M.options ={
  relativenumber = true,
  nvChad ={
    insert_nav = true
  }
}

M.ui = {
   theme = "gruvchad",
}


-- Install plugins
local userPlugins = require "custom.plugins" -- path to table

M.plugins = {
  --install = userPlugins,
  user = userPlugins,
  status={
    alpha=true,
   -- vim_matchup=true,
   -- nvimtree=false
  },
  options={
    packer ={
    --  init_file = "plugins.packerInit"
    },
  }

}

M.mappings ={
-- navigation in insert mode, only if enabled in options
 insert_nav = {
    --backward = "<C-h>",
--    end_of_line = "<C-e>",
--    forward = "<C-f>",
--    next_line = "<C-n>",
--    prev_line = "<C-p>",
--    beginning_of_line = "<C-a>",
 },
terminal ={
 --esc_termmode={ "<esc>"}
--t={"<esc>","<C-\\><C-n>"}
}

}
```
still have some problem

**Reply by WingDust** [reply]
@siduck

**Reply by siduck** [reply]
u commented the t=( .. part so why would it work bruh

**Reply by WingDust** [reply]
@siduck I don't how to write a specific and right override mapping

**Reply by siduck** [reply]
@WingDust you need to learn it then, ig you are new to nvchad. please read the docs from start

**Reply by Patrick-web** [reply]
Thanks

**Reply by azikdinal** [reply]
you gotta be genius man

**Reply by siduck** [reply]
@azikdinal  C-x is just mapped to this 
![image](https://github.com/NvChad/NvChad/assets/59060246/28bcd114-7075-4b58-8a91-1039faae6324)


So  Ctrl+\ + Ctrl+n is the default vim mapping

**Reply by azikdinal** [reply]
Switched to nvim a few days ago. Don't know a lot of features yet. But writing-editing text became exiting with no need to touch mouse. 

Thanks, man.

**Reply by floresce** [reply]
How come this doesn't show up under :Telescope keymaps?

**Reply by siduck** [reply]
@Floresce i think its because its in "Terminal" mode
